src/main/java/service/ShopService.java

```diff
@@ -0,0 +1,7 @@
+package service;
+
+public interface ShopService {
+    boolean execProductTransaction(String productname,
```

Comment:
```suggestion
    boolean executeProductTransaction(String productname,
```

погана практика використовувати скорочення в назвах методів